build.gradle

```diff
@@ -195,6 +206,24 @@ project(':aeron-samples') {
     build.doLast {
         tasks.shadowJar.execute()
     }
+
```

Comment:
I have already moved all of this into the build.gradle in aeron-tools. I also create a separate tools.jar which packages everything together just like samples.jar.